lib/datadog/tracing/contrib/karafka/integration.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../integration'
+require_relative 'patcher'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Karafka
+        # Description of Kafka integration
+        class Integration
+          include Contrib::Integration
+
+          MINIMUM_VERSION = Gem::Version.new('2.1.0')
+
+          # @public_api Changing the integration name or integration options can cause breaking changes
+          register_as :karafka, auto_patch: false
+
+          def self.version
+            Gem.loaded_specs['karafka'] && Gem.loaded_specs['karafka'].version
+          end
+
+          def self.loaded?
+            !defined?(::Karafka).nil?
+          end
+
+          def self.compatible?
+            super && version >= MINIMUM_VERSION
```

Comment:
please note how I do versioning: https://karafka.io/docs/Versions-Lifecycle-and-EOL/#versioning-strategy

Reply:
I don’t think simply picking the latest or an updated version should be the goal of this compatibility check. Ideally, the `minimum_version` should be the version that supports the specific events we want to subscribe to. Could you suggest one that fits this criteria? I can use 2.0.x, but those versions seem too outdated and 2.1.0 seems fit to me.

Reply:
happy to start with 2.2

Reply:
Good with me!